x-pack/packages/kbn-cloud-security-posture-common/utils/show_error_toast.ts

```diff
@@ -6,7 +6,7 @@
  */
 import type { CoreStart } from '@kbn/core/public';
 import { i18n } from '@kbn/i18n';
-import { extractErrorMessage } from '../../../common/utils/helpers';
+import { extractErrorMessage } from './helpers';
 
```

Comment:
shouldn't this file be in `kbn-cloud-security-posture` instead? A toast notification It's mainly a frontend function, not meant to be shared with the server.